dynamoDBtoCSV.js

```diff
@@ -32,31 +32,33 @@ program
   .option("-s, --size [size]", "Number of lines to read before writing.", 5000)
   .parse(process.argv);
 
-if (!program.table) {
+  const options = program.opts();
+
+if (!options.table) {
   console.log("You must specify a table");
-  program.outputHelp();
+  options.outputHelp();
```

Comment:
I don't think that's correct. options is doesn't have an outputHelp method. it's a program.OptionValue